moveit_ros_planning failing to build on indigo

http://jenkins.ros.org/view/IbinS32/job/ros-indigo-moveit-ros-planning_binarydeb_saucy_i386/69/console

It looks like a missing dependency declaration. 

```
-- Using these message generators: gencpp;genlisp;genpy
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.26") 
-- checking for module 'eigen3'
--   found eigen3, version 3.2.0
-- Found Eigen: /usr/include/eigen3  
-- Eigen found (include: /usr/include/eigen3)
CMake Error at /usr/share/cmake-2.8/Modules/FindQt4.cmake:1382 (message):
  Found unsuitable Qt version "" from NOTFOUND, this code requires Qt 4.x
Call Stack (most recent call first):
  CMakeLists.txt:25 (find_package)


-- Configuring incomplete, errors occurred!
dh_auto_configure: cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCATKIN_BUILD_BINARY_PACKAGE=1 -DCMAKE_INSTALL_PREFIX=/opt/ros/indigo -DCMAKE_PREFIX_PATH=/opt/ros/indigo returned exit code 1
make[1]: *** [override_dh_auto_configure] Error 2
```

--- a/planning/package.xml
+++ b/planning/package.xml
@@ -24,6 +24,7 @@
   <build_depend>dynamic_reconfigure</build_depend>
   <build_depend>angles</build_depend>
   <build_depend>cmake_modules</build_depend>
+  <build_depend>libqt4-dev</build_depend>
 
   <run_depend>moveit_core</run_depend>
   <run_depend>moveit_ros_perception</run_depend>
@@ -31,6 +32,7 @@
   <run_depend>actionlib</run_depend>
   <run_depend>dynamic_reconfigure</run_depend>
   <run_depend>angles</run_depend>
+  <run_depend>libqt4</run_depend>
 
   <export>
     <moveit_core plugin="${prefix}/planning_request_adapters_plugin_description.xml"/> 

